Makefile

```diff
@@ -293,7 +293,18 @@ test-db-migration: start-docker ## Gets diff of upgrade vs new instance schemas.
 	./scripts/mysql-migration-test.sh
 	./scripts/psql-migration-test.sh
 
-test-server: start-docker go-junit-report do-cover-file ## Runs tests.
+gomodtidy:
+	@cp go.mod go.mod.orig
+	@cp go.sum go.sum.orig
+	$(GO) mod tidy
+	@if [ "$$(diff go.mod go.mod.orig)" != "" -o "$$(diff go.sum go.sum.orig)" != "" ]; then \
+		echo "go.mod/go.sum was modified. diff- $(diff go.mod go.mod.orig) Run \"go mod tidy\"."; \
+		rm go.*.orig; \
+		exit 1; \
+	fi;
+	@rm go.*.orig;
```

Comment:
Maybe add `go.mod.orig` and `go.mod.orig` to `gitignore` just to make sure no one commits it?